_data/authors.yml

```diff
@@ -314,7 +314,7 @@ tim_vierbergen:
   title: Software Engineer
   title2: Principal Consultant & Competence Lead CI/CD
   email: tim.vierbergen@ordina.be
-  bio: Tim is a senior developer and architect at Jworks Ordina Belgium. Tim has a DevOps culture mindset and is experienced in many different domains. From frontend to backend to pipelining and automation. Tim is keen on learning new technologies.<br /><br />Tim is a true sportsman and spends part of his free time running and working out. Tim is also passionate about surfing.
+  bio: Tim is a senior developer and architect at Jworks Ordina Belgium. Tim has a DevOps culture mindset and is experienced in many different domains. From frontend to backend to pipelining and automation. Tim is keen on learning new technologies.<br /><br />Tim is a true sportsman. Tim spends part of his free time running and working out and is also passionate about surfing.
```

Comment:
JWorks
Maybe a bit too much "Tim is/has"? Perhaps try to combine some of the sentences.